features/reporting.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-10-21
 name: Reporting API
-description: TODO
+description: The Reporting API is a generic way to make reports available based on various platform features in a consistent manner, such as information about Content Security Policy, Permissions-Policy, or feature deprecations.
 spec: https://w3c.github.io/reporting/
 compat_features:
```

Comment:
Should this BCD key (and sub-keys) be listed here too: api.SecurityPolicyViolationEvent?

Reply:
Those keys are in the CSP feature. The Reporting API is topic-agnostic, so topic-specific things probably have a better home w/ their topic area.